src/Features/CSharp/Portable/CodeFixes/QualifyMemberAccess/CSharpQualifyMemberAccessCodeFixProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeFixes.Qualify;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.QualifyMemberAccess
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.QualifyMemberAccess), Shared]
+    [ExtensionOrder(After = PredefinedCodeFixProviderNames.RemoveUnnecessaryCast)]
+    internal class CSharpQualifyMemberAccessCodeFixProvider : AbstractQualifyMemberAccessCodeFixprovider<SimpleNameSyntax>
+    {
+        protected override string GetId() => CSharpFeaturesResources.AddThisQualification;
```

Comment:
I would call this "Qualify member access".  It will be clear from the preview that that's done by adding "this.".  I don't like messages/titles that literally state exactly what the code change is.  that feels redundant.

Bonus: by doing this, you won't need another title if we add static qualification.